packages/cli/src/lib/collect/collect-command.unit.test.ts

```diff
@@ -51,6 +51,32 @@ describe('collect-command', () => {
         persist: expect.objectContaining({
           filename: 'my-report',
           outputDir: '/test',
+          format: ['json'],
+        }),
+      }),
+    );
+  });
+
+  it('should call collect and use terminal arguments', async () => {
+    await yargsCli(
+      [
+        'collect',
+        '--verbose',
+        '--config=/test/code-pushup.config.ts',
+        '--persist.format=md',
+      ],
+      {
+        ...DEFAULT_CLI_CONFIGURATION,
+        commands: [yargsCollectCommandObject()],
+      },
+    ).parseAsync();
+
+    expect(collectAndPersistReports).toHaveBeenCalledWith(
+      expect.objectContaining({
+        verbose: true,
+        config: '/test/code-pushup.config.ts',
+        persist: expect.objectContaining({
+          format: ['md'],
```

Comment:
```suggestion
```
This test is unnecessary. It tests the same properties and a subset of functions tested in the test above.